python/dolfin/__init__.py

```diff
@@ -31,13 +31,15 @@
 # Import cpp modules
 from .cpp import __version__
 
-from .cpp.common import (Variable, has_debug, has_hdf5, has_scotch,
-                         has_hdf5_parallel, has_mpi4py, has_petsc_complex,
-                         has_petsc4py, has_parmetis, has_slepc, has_slepc4py,
-                         git_commit_hash, DOLFIN_EPS, DOLFIN_PI, TimingType,
-                         timing, timings, list_timings)
 
-from .cpp import MPI
+from dolfin.common import (
+    Variable, has_debug, has_hdf5, has_scotch, has_hdf5_parallel,
+    has_mpi4py, has_petsc_complex, has_petsc4py, has_parmetis,
+    has_slepc, has_slepc4py, git_commit_hash, DOLFIN_EPS, DOLFIN_PI,
```

Comment:
I don't think we should expose `DOLFIN_EPS` and `DOLFIN_PI`. `math.pi`, etc are available.

Reply:
DOLFIN_EPS is scattered all over demos and tests. What should I replace it with?

Reply:
Eventually https://docs.scipy.org/doc/numpy/reference/generated/numpy.finfo.html.

How about just adding a comment for now that it might be removed?